### PR TITLE
Working tests and provider stats table.

### PR DESCRIPTION
**Pull request by mannickutd**
Fix for tests and import error.

### CODE REVIEW

File: app/schemas.py [hunk 1]
```diff
@@ -58,6 +48,25 @@ def make_instance(self, data):
         return super(BaseSchema, self).make_instance(data)
 
 
+class CreatedBySchema(ma.Schema):
```
Comment (ashimali) [inline app/schemas.py]: May as well at this to ApiKey as well

File: tests/app/dao/test_templates_dao.py [hunk 2]
```diff
@@ -11,12 +11,13 @@
 import pytest
 
 
-def test_create_template(sample_service):
+def test_create_template(sample_service, sample_user):
```
Comment (ashimali) [inline tests/app/dao/test_templates_dao.py]: Probably add an assert that template history record exists

**Comment by ashimali** [comment]
You might want to add this to get_service_and_api_key_history in /service/rest.py and surface in admin app so Pete/Henry can sign off story.

File: app/schemas.py (hunk 1 above)
Comment (mannickutd) [inline app/schemas.py]: Done

File: tests/app/dao/test_templates_dao.py (hunk 2 above)
Comment (mannickutd) [inline tests/app/dao/test_templates_dao.py]: Separate tests added.

**Comment by ashimali** [comment]
👍

**Pull request merged**